atd-vze/src/views/Crashes/CrashChangeLog.js

```diff
@@ -187,7 +197,7 @@ class CrashChangeLog extends Component {
       content = <p>No changes found for this record.</p>;
     } else {
       modal = (
-        <Modal isOpen={this.state.modal} className={this.props.className}>
+        <Modal isOpen={this.state.modal} className="mw-100 mx-5">
```

Comment:
There is a `modal-xl` class in Bootstrap 4 but it wasn't super wide so I made this take up the whole width and added some margin prevent it from rendering all the way to the edge.